app/models/document.rb

```diff
@@ -239,7 +239,7 @@ def publish!
         email_alert_api.send_alert(EmailAlertPresenter.new(self).to_json)
       end
 
-      publish_request.code == 200 && rummager_request.code == 200
+      publish_request.code == 200 && rummager_request.code == 202
```

Comment:
That does indeed seem like a very useful thing, thanks for sharing @elliotcm!  Glad this sort of stuff is on a roadmap, and for the avoidance of doubt I'm totally onboard with the suggested fix of removing the status code checks.